docs/usage/polling.md

```diff
@@ -1,16 +1,15 @@
 # Polling Legacy Z-Wave Devices
 
-Some legacy devices don't report all of their values automatically and require polling to automatically update their values. Some platforms, such as Smartthigns and Open Z-Wave, poll such devices automatically whether you need them polled or not. Because such polling can have detrimental effects on your mesh network, zwave-js does not poll devices by default. Polling can quickly lead to network congestion and should be used very sparingly and only where necessary.
+Some legacy devices don't report all of their values automatically and require polling to automatically update their values. Some platforms, such as Smartthigns and Open Z-Wave, poll such devices automatically whether you need them polled or not. Because such polling can have detrimental effects on your mesh network, Z-Wave JS does not poll devices by default. Polling can quickly lead to network congestion and should be used very sparingly and only where necessary.
 
-zwavejs2mqtt allows you to configure scheduled polling on a per-value basis, which you can use to keep only the values that you need updated.
-It also allows you to poll individual values on-demand from your automations, which should be preferred over blindly polling all of the time, if possible.
+Zwavejs2Mqtt allows you to configure scheduled polling on a per-value basis, which you can use to keep only the values that you need updated. It also allows you to poll individual values on-demand from your automations, which should be preferred over blindly polling all of the time, if possible.
 
 > [!NOTE]
-> Many values can only be polled together. For example `targetValue`, `currentValue` and `duration` for most of the switch-type CCs. Before enabling polling for all values of a node, users should check whether polling a single value already updates the desired other values too.
+> Sometimes some values can only be polled together as a group. For example `targetValue`, `currentValue` and `duration` must be polled together for most of the switch-type CCs. Before enabling polling for all values of a node, you should check whether polling a single value already updates the other desired values too.
```

Comment:
This isn't clear. It's not that they **must** be polled together, you **cannot** poll one without the other. Polling multiple values of the same group **can** result in unnecessary commands (not sure about the last part).

Reply:
How would users check whether polling one value also polls the others, so they can avoid this problem? Do they have to inspect the driver source code? Read the Z-Wave specification? Watch logs and experimentally determine it?

Reply:
@AlCalzone Is it that they have to be polled at once, or that polling one returns the other related values?

Reply:
RE @kpine comment, is my update the best way?

Reply:
Works for me.